### PR TITLE
Android invite recommendation implementation

### PR DESCRIPTION
**Pull request by mhrfky**
I have added the recommendation system in the project invite. To reduce the confusion and to give the user more control, I have created a seperate part to manage one's invitations. With that if one has already invited first 10 people, he does not have scroll down to invite others. All sent invitations can be handled in a seperate part.

image

image

image

### CODE REVIEW

File: android/app/src/main/java/com/bounswe2020group3/paperlayer/invite/InvitePresenter.kt [hunk 1]
```diff
@@ -29,12 +29,41 @@ class InvitePresenter @Inject constructor(private var model: InviteContract.Mode
 
     override fun subscribeAuthToken() {
         val userProfileSub = model.getAuthToken().subscribe { token ->
-            fetchAllUsers(token.id)
+            //fetchAllUsers(token.id)
```
Comment (yusufbayam) [inline android/app/src/main/java/com/bounswe2020group3/paperlayer/invite/InvitePresenter.kt:32]: Can you remove comment lines please?

File: android/app/src/main/java/com/bounswe2020group3/paperlayer/invite/ManageInvitesPresenter.kt [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+    override fun unbind() {
+        super.unbind()
+        disposable.clear()
+
+    }
+
+    override fun subscribeAuthToken() {
+        val userProfileSub = model.getAuthToken().subscribe { token ->
+            //fetchAllUsers(token.id)
```
Comment (yusufbayam) [inline android/app/src/main/java/com/bounswe2020group3/paperlayer/invite/ManageInvitesPresenter.kt:91]: Can you remove comment lines please?

**Review by yusufbayam — COMMENTED** [review]
Looking good, we can merge after resolving issues

File: android/app/src/main/java/com/bounswe2020group3/paperlayer/util/Constants.kt [hunk 3]
```diff
@@ -1,5 +1,5 @@
 package com.bounswe2020group3.paperlayer.util
 
-//const val BASE_URL = "https://paperlayer.herokuapp.com"
-const val BASE_URL = "https://paperlayer-backend.azurewebsites.net"
+const val BASE_URL = "https://paperlayer.herokuapp.com"
+//const val BASE_URL = "https://paperlayer-backend.azurewebsites.net"
```
Comment (ybedirhanpak) [inline android/app/src/main/java/com/bounswe2020group3/paperlayer/util/Constants.kt:4]: I think we shouldn't keep heroku in the master anymore. Let's just use it on local and never commit it

**Review by ybedirhanpak — COMMENTED** [review]
Needs solving the conflicts @mhrfky, after that ready to merge

**Review by yusufbayam — APPROVED** [review]
Looking good

**Pull request merged**